src/App.js

```diff
@@ -32,15 +33,69 @@ function App() {
       job: "github",
     },
   };
+  const [apiData, setApiData] = useState(UserData);
```

Comment:
The default `UserData` we had here was a good proof of concept for us, it showed us what the data would look like on render for the UI PR. But essentially, we do not need this. It is extra code for us. 

On initial render, we do not have any data, therefore our `apiData` will be an empty object. 

Whether data is available or not will be handled in the respective components that need this data i.e `UserDisplay` will be responsible for deciding whether to render the user's avatar or the octocat. 

```suggestion
  const [apiData, setApiData] = useState(UserData);
    const [apiData, setApiData] = useState({});

```